tlair/src/main/scala/at/forsyte/apalache/tla/lir/io/PrettyWriter.scala

```diff
@@ -32,24 +33,42 @@ class PrettyWriter(writer: PrintWriter, textWidth: Int = 80, indent: Int = 2) ex
   // the stack of lambda declarations
   private var lambdaStack: List[TlaOperDecl] = Nil
 
-  def write(mod: TlaModule): Unit = {
-    writer.write(pretty(toDoc(mod), textWidth).layout)
-  }
+  def prettyWriteDoc(doc: Doc): Unit = writer.write(pretty(doc, textWidth).layout)
 
-  def write(decl: TlaOperDecl): Unit = {
-    writer.write(pretty(toDoc(decl), textWidth).layout)
-  }
+  def blockComment(commentStr: String): Doc = text(s"(* $commentStr *)") <> line
+  def lineComment(commentStr: String): Doc = text(s"\\* $commentStr") <> line
```

Comment:
Why do we have two methods? If you need both, introduce an option on how to output it. But I would just always write comments in one style, e.g., as block comments.

Reply:
The short answer to this, plus all of the questions about `nDashes` and `nEquals` is that a lot of the tests compare output against hard-coded strings (file contents). It was either this, or rewriting all of the test files' formatting.

Reply:
I don't think it is a good approach to create technical debt because of the bad tests ;-)